concepts/algorithms/covering_edges.py

```diff
@@ -0,0 +1,28 @@
+"""Covering Edges
+
+cf. Carpineto, Claudio, and Giovanni Romano.
+Concept data analysis: Theory and applications.
+John Wiley & Sons, 2004.
+"""
+
+
+def covering_edges(concept_list, context):
```

Comment:
Can you add a doctest?

Reply:
Yes, no problem :)

Reply:
Thanks. Getting ready. Can you try type annotations as well? :)

Reply:
Not used to write these sorry :) Good practise for me.